### PR TITLE
Update doc with sync command

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: docs/en/structured-data/catalog-service.md [hunk 1]
```diff
@@ -87,15 +90,15 @@ This command maps the hive database `default` into a database in Alluxio called
...
-    --option hive.metastore.uris=thrift://metastore_host:9083
+$ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
+    thrift://metastore_host:9083 default
 ```
 
 > **Note:** When databases are attached, all tables will be synced from the configured UDB.
 If out-of-band updates occur to the database or table and the user wants query results to reflect
 the updates, the user must detach ([See Detaching Databases](#detaching-databases)) and then
-re-attach the database to reflect up any updates.
+re-attach the database to reflect up any updates. Alternatively, if the only changes to the database
```
Comment (ZacBlanco) [inline docs/en/structured-data/catalog-service.md]: new sentence should be on a newline

https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html#line-breaks

Comment (yuzhu) [reply docs/en/structured-data/catalog-service.md]: done

**Comment by ZacBlanco** [comment]
failed tests:
```
Tests in error: 
alluxio.client.cli.fs.JobServiceFaultToleranceShellTest.distributedMv(alluxio.client.cli.fs.JobServiceFaultToleranceShellTest)
  Run 1: JobServiceFaultToleranceShellTest.before:52 » IO java.lang.RuntimeException: a...
  Run 2: JobServiceFaultToleranceShellTest.before:52 » IO java.lang.RuntimeException: a...
```
since this is a doc-only change I'm going to merge

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**